OpenRA.Game/Server/Connection.cs

```diff
@@ -16,14 +16,14 @@
 namespace OpenRA.Server
 {
 	public class Connection
-	{
+    {
+        public const int MaxOrderLength = 131072;
```

Comment:
You should change your IDE project settings to use tabs instead of spaces for OpenRA. We support http://editorconfig.org/